reframe/frontend/loader.py

```diff
@@ -203,6 +203,7 @@ def load_all(self, force=False):
         for d in self._load_path:
             getlogger().debug(f'Looking for tests in {d!r}')
             if not os.path.exists(d):
+                getlogger().warning(f"check path {d!r} doesn't exist")
```

Comment:
Do not use abbreviated forms in error messages. I will fix this and merge it.

```suggestion
                getlogger().warning(f'check path {d!r} does not exist')
```